modules/chromomap.nf

```diff
@@ -0,0 +1,102 @@
+process chromomap {
+      publishDir "${params.output}/${name}/chromomap", mode: 'copy', pattern: "sample_overview.html"
+      label 'chromomap'
+      errorStrategy 'retry'
+      maxRetries 3
+    input:
+      tuple val(name), file(chromosome), file(annotation)
```

Comment:
please start using path instead of file
* path handles missing files better as it gives a clear file missing error code instead of creating a file